### ISSUE TITLE
order_by_type setting default switched from True to False?

### ISSUE THREAD

**Issue by timgraham** [timothycrosley/isort#323]
In 48718ce5cc6303d16a642677306bf196001ed196, the default value of `order_by_type` was switched from `True` to `False`. Was it intentional? If so, I didn't see any indication of it in the change log.

This change seems to be backwards incompatible as it causes isort to suggest changes like this:
```diff
diff --git a/django/apps/registry.py b/django/apps/registry.py
index 46c10ef..d31009a 100644
--- a/django/apps/registry.py
+++ b/django/apps/registry.py
@@ -1,7 +1,7 @@
 import sys
 import threading
 import warnings
-from collections import Counter, OrderedDict, defaultdict
+from collections import Counter, defaultdict, OrderedDict
 from functools import partial
 
 from django.core.exceptions import AppRegistryNotReady, ImproperlyConfigured
```

**Comment by timothycrosley** [comment]
Hi @timgraham,

Thanks for pointing this out! This was certainly not intentional, I've created and pushed a hot fix release `4.1.2` to quickly resolve this issue.

Thanks!

~Timothy

**Comment by timgraham** [comment]
Thank you for the quick resolution.
